### PR TITLE
Mobile version -- portfolio completion

### PR DESCRIPTION
**Pull request by yosefHesham**
Changes:

- Added works section
- Added about me section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,58 +1,161 @@
...
+            <li class="items">
+              <a href="#"> <img src="assets/images/github.svg" alt="" /> </a>
+            </li>
+            <li class="items">
+              <a href="#"> <img src="assets/images/angellist.svg" alt="" /> </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="recent-works">
```
Comment (micheaol) [inline index.html:53]: Please ensure that the container of the projects has a `display:grid` according to the project requirement.

File: index.html [hunk 2]
```diff
@@ -1,58 +1,161 @@
...
+              </li>
+              <li><p class="lang">javScript</p></li>
+              <li>
+                <p class="lang">html</p>
+              </li>
+            </ul>
+            <a href="#" class="button"> See Projects </a>
+          </div>
+        </article>
+        <article class="card">
```
Comment (micheaol) [inline index.html:75]: According to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468), the project is five (5) and you have three implemented in your design. Kindly implement the remaining two to conform with the design.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @yosefHesham , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (yosefHesham) [reply index.html:53]: I really forgot that , thank you

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @yosefHesham , 

Great work on making the changes requested by the previous reviewer 👏 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**